### PR TITLE
header footer components created and tested

### PR DESCRIPTION
**Pull request by Louie-cpu**

### CODE REVIEW

**Comment by Louie-cpu** [comment]
https://trello.com/c/GKsw2rhw

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+  render(
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  )
+  const bankLogo = screen.getByAltText("bank buddy logo")
+  const DashboardLink = screen.getByText("Dashboard")
+  const HomeLink = screen.getByText("Home")
+  const SignInLink = screen.getByText("Sign In")
+  const FAQLink = screen.getByText("FAQ")
```
Comment (sjproctor) [inline src/App.test.js]: make sure your variables are in camelCase here!

File: src/__tests__/Header.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  expect(DashboardLink).toBeInTheDocument()
+  expect(DashboardLink.getAttribute("href")).toBe("/dashboard")
+
+  const HomeLink = screen.getByText(/Home/i)
+  expect(HomeLink).toBeInTheDocument()
+  expect(HomeLink.getAttribute("href")).toBe("/")
+
+  const FAQLink = screen.getByText(/FAQ/i)
+  expect(FAQLink).toBeInTheDocument()
+  expect(FAQLink.getAttribute("href")).toBe("/FAQ")
```
Comment (sjproctor) [inline src/__tests__/Header.test.js]: same thing with variables here.

**Review by sjproctor — COMMENTED** [review]
just a couple small things.

**Review by sjproctor — APPROVED** [review]

**Pull request merged**